com.github.emmanueltouzery.cigale.desktop

```
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Comment=
```

```suggestion
Comment=
```
I guess you forgot a comment here

---

not really. I'll just remove the Comment= line